pythia/seo/ELO.py

```diff
@@ -0,0 +1,195 @@
+        self.max_comparisions = max_comparisons
+        self.score_memory = score_memory
+
+        self.get_data(delimiter)
+
+    def get_data(self, delimiter):
+        """
+        Loads the data from the Disk and prepares the Ranking DataFrame.
+
+        Parameters
+        ----------
+        delimiter : str
+            Delimiter for reading the CSV files
+        """
+        self.classifications = pd.read_csv(self.classifications, delimiter=delimiter)
+        self.rankings = pd.read_csv(self.rankings, delimiter=delimiter)
+        self.rankings.set_axis(self.rankings['image_id'], inplace=True)
+        self.rankings['score'] = self.default_score
+        self.rankings['k_value'] = self.k_value
+        self.rankings['count'] = 1
+        self.rankings['std_dev'] = self.k_value
+        self.rankings['last_scores'] = str(self.default_score)
+
+    def expected_score(self, image_0, image_1):
+        """
+        Given two AR scores, calculates expected probability of `image_0` being more complex.
+
+        Parameters
+        ----------
+        image_0 : int
+            Image id for first image
+        image_1 : int
+            Image id for second image
+
+        Returns
+        -------
+        expexted_0_score : float
+            Expected probability of `image_0` being more complex.
+        """
+        expexted_0_score = 1.0 / (1.0 + 10 ** ((self.rankings.loc[image_1]['score'] -
+                                                self.rankings.loc[image_0]['score']) / 400.00))
+        return expexted_0_score
+
+    def new_rating(self, image, score_for_image, image_expected_score):
+        """
+        Calculates new rating based on the ELO algorithm.
+
+        Parameters
+        ----------
+        image : int
+            Image id for the image
+        score_for_image : int
+            Actual result of classification of the image in a pairwise match.
+            `0` denotes less complex, `1` denotes more complex
+        image_expected_score : float
+            Expected result of classification of image in a pairwise match
+            based on current rating of the image.
+
+        Returns
+        -------
+        new_image_rating : float
+            New rating of image after the classification match.
+        """
+        new_image_rating = self.rankings.loc[image]['score'] + self.rankings.loc[image]['k_value'] * (score_for_image - image_expected_score)
+        return new_image_rating
+
+    def score_update(self, image_0, image_1, score_for_image_0):
+        """
+        Updates the ratings of the two images based on the complexity classification.
+
+        Parameters
+        ----------
+        image_0 : int
+            Image id for first image
+        image_1 : int
+            Image id for second image
+        score_for_image_0 : int
+            Actual result of classification of the image 0 in a pairwise match.
+            `0` denotes less complex, `1` denotes more complex
```

Comment:
would it make more sense to use a boolean instead? And the argument being: `wins_0_over_1` or simpler: `wins_0`